tests/core/test_indexset.py

```diff
@@ -2,9 +2,10 @@
 import pandas.testing as pdt
 import pytest
 
-from ixmp4.core import IndexSet, Platform
+import ixmp4
+from ixmp4 import IndexSet
```

Comment:
If we import ixmp4 as a whole, we don't really need to import `IndexSet` separately, right? 
Also, importing that from `ixmp4.core` was intentional since @danielhuppmann suggested to clean up the top-level namespace in the future, so please keep that if we keep the explicit import.

Reply:
We do this in every core test as far as i can see. I prefer it this way since: 
```py
except ixmp4.IndexSet.NotFound: 
    pass
```
is a little long.

Reply:
I'm generally fine with the shorter version. We'll have to rework these imports if we remove stuff from the top level namespace, but maybe best to avoid preemptive obedience.

Reply:
Hmm, I would argue we need to think about #84 and what exactly we want in the top level namespace first.
I dont agree that its the best to ONLY have `Platform` in the root ixmp4 module. These 'index' type modules do give us the opportunity to bundle things that one might need together in ixmp4. 
I think from a users' perspective importing ixmp4 and then being able to pretty much type hint anything and catch any exception you might encounter is pretty useful. If only `Platform` resides in `ixmp4` then we might as well abandon it in favor of `ixmp4.Platform` completely. I've adjusted the imports to come from `ixmp4.core` now and we can pick up in #84 and a follow-up PR.